app/portainer/components/forms/stack-redeploy-git-form/stack-redeploy-git-form.controller.js

```diff
@@ -96,19 +96,20 @@ class StackRedeployGitFormController {
       '<be-feature-indicator feature="stackPullImageFeature"></be-feature-indicator></div></div>';
     const template = angular.element(tplCrop);
     const html = this.$compile(template)(this.$scope);
-    this.ModalService.confirmStackUpdate(html, true, true, 'btn-warning', function (result) {
+    var that = this;
+    this.ModalService.confirmStackUpdate(html, true, true, 'btn-warning', async function (result) {
```

Comment:
easier would be just to use an arrow function, then `this` doesn't change
```suggestion
    this.ModalService.confirmStackUpdate(html, true, true, 'btn-warning', async (result) => {
```

Reply:
I have updated code according to your suggestion. :)